Tests/ClientTests/Mocks/MockBrowserViewController.swift

```diff
@@ -6,24 +6,72 @@ import UIKit
 @testable import Client
 
 class MockBrowserViewController: BrowserViewController {
-    var switchToPrivacyModeCalled: (Bool) -> Void = { _ in }
-    var switchToTabForURLOrOpenCalled: (Bool) -> Void = { _ in }
-    var openBlankNewTabCalled: (Bool) -> Void = { _ in }
-    var handleQueryCalled: (String) -> Void = { _ in }
+    var switchToPrivacyModeCalled = false
+    var switchToPrivacyModeIsPrivate = false
+    var switchToTabForURLOrOpenCalled = false
+    var switchToTabForURLOrOpenURL: URL?
+    var switchToTabForURLOrOpenUUID: String?
+    var switchToTabForURLOrOpenIsPrivate = false
+
+    var openBlankNewTabCalled = false
+    var openBlankNewTabFocusLocationField = false
+    var openBlankNewTabIsPrivate = false
+    var openBlankNewTabSearchText: String?
+
+    var handleQueryCalled = false
+    var handleQuery: String?
+    var showLibraryCalled = false
+    var showLibraryPanel: LibraryPanelType?
+
+    var openURLInNewTabCalled = false
+    var openURLInNewTabURL: URL?
+    var openURLInNewTabIsPrivate = false
+
+    var switchToPrivacyModeCount = 0
+    var switchToTabForURLOrOpenCount = 0
+    var openBlankNewTabCount = 0
+    var handleQueryCount = 0
+    var showLibraryCount = 0
+    var openURLInNewTabCount = 0
 
     override func switchToPrivacyMode(isPrivate: Bool) {
-        switchToPrivacyModeCalled(true)
+        switchToPrivacyModeCalled = true
```

Comment:
nit, the called property is accounted for when the count is 1 or more. We can chat over this and merge this PR in the meantime